xml/generic-entities.ent

```diff
@@ -0,0 +1,532 @@
+<!ENTITY xen                    "Xen">
+<!ENTITY xenreg                 "Xen&reg;">
+<!ENTITY xenstore               "XenStore">
+
+
+
+
+<!-- DAEMONS -->
+
+<!ENTITY chronyd                "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>chronyd</systemitem>">
+<!ENTITY crond                  "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>cron</systemitem>">
+<!ENTITY firewalld              "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>firewalld</systemitem>">
+<!ENTITY libvirtd               "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>libvirtd</systemitem>">
+<!ENTITY oprofd                 "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>oprofile</systemitem>">
+<!ENTITY systemd                "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>systemd</systemitem>">
+<!ENTITY systemdcd              "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>systemd-coredump</systemitem>">
+<!ENTITY tuned                  "<systemitem xmlns='http://docbook.org/ns/docbook' class='daemon'>tuned</systemitem>">
+
+
+
+<!-- APPLICATIONS -->
+
+<!ENTITY clamav                 "ClamAV">
+<!ENTITY cpufreq                "CPUfreq">
+<!ENTITY gpg                    "GPG"> <!-- use GnuPG instead? -->
+<!ENTITY mariadb                "MariaDB">
+<!ENTITY nautilus               "&gnome; Files">
+<!ENTITY oprof                  "OProfile">
+<!ENTITY postgresql             "PostgreSQL">
+<!ENTITY powertop               "powerTOP">
+<!ENTITY stap                   "SystemTap">
+
+
+
+<!-- HARDWARE TECHNOLOGIES -->
+
+<!ENTITY nvme                   "NVMe">
+<!ENTITY nvmeof                 "NVMe over Fabric">
+<!ENTITY fcnvme                 "FC-NVMe">
+<!ENTITY dmmpio                 "DM-Multipathing">
+<!ENTITY mpio                   "multipath I/O">
+
+<!ENTITY hdmi                   "HDMI">
+<!ENTITY hdmireg                "&hdmi;*">
+<!ENTITY mipireg                "MIPI*">
+<!ENTITY mipidsireg             "&mipireg;&nbsp;DSI*">
+<!ENTITY openglreg              "OpenGL*">
+<!ENTITY openglesreg            "&openglreg;&nbsp;ES">
+
+<!ENTITY microsd                "microSD">
+<!ENTITY microsdhc              "microSDHC">
+<!ENTITY bt                     "Bluetooth">
+<!ENTITY btreg                  "Bluetooth*">
+
+<!ENTITY usbflashdrive          "USB flash drive">
+<!ENTITY usbflashdrivecap       "USB Flash Drive">
+
+
+<!-- NETWORK PROTOCOLS -->
+<!ENTITY ptp             "Precision Time Protocol">
+<!ENTITY ntp             "Network Time Protocol">
+
+
+<!-- MISCELLANEOUS -->
+
+<!ENTITY iscsi                  "iSCSI">
+<!ENTITY netteam                "Network Teaming">
+<!ENTITY openscap               "OpenSCAP">
+<!ENTITY pcidss                 "Payment Card Industry Data Security Standard">
+<!ENTITY pcidssa                "PCI DSS">
+<!ENTITY uefisecboot            "UEFI Secure Boot">
+
+
+
+<!-- BOOK NAMES -->
```

Comment:
JeOS Quick Start (or whatever it will be called in the future) is missing here

Reply:
True. It is titled this way, without use of an entity:
https://github.com/SUSE/doc-sle/blob/78936f5b8a1b76ecaab8ec9ef76156243e8e7562/xml/art_jeos.xml#L11

It's not the only one:
```
> for f in xml/*; do [[ `grep -nP -C1000 '<(book|set|article)' $f` ]] && grep -m1 '<title' $f; done
 <title>Podman Guide</title>
 <title>&admin;</title>
 <title>AMD Secure Encrypted Virtualization (AMD-SEV) Guide</title>
 <title>&instquick;</title>
 <title>&instquick;</title>
  <title>&jeos; Quick Start Guide</title><productname>&sls; JeOS</productname>
 <title>&modulesquick;</title>
 <title>&nvidia; virtual GPU for &kvm; guests</title>
 <title>&pcidss; (&pcidssa;) Guide</title>
  <title>&rpiquick;</title>
 <title>Virtualization Best Practices</title>
  <title>&admin;</title>
  <title>&ayguide;</title><productname>&productname;</productname>
  <title>Container Guide</title>
  <title>&deploy;</title><productname>&productname;</productname>
  <title>&deploy;</title>
  <title>&gnomeuser;</title>
  <title>&reference;</title><productname>&productname;</productname>
  <title>&rmtool; Guide</title>
  <title>&secguide;</title>
  <title>&startup;</title><productname>&productname;</productname>
  <title>&storage_guide;</title>
  <title>&tuning;</title>
  <title>&upgrade_guide;</title><productname>&productname;</productname>
  <title>&virtual;</title>
<title>&productname; Documentation</title>
 <title>&productname; Documentation</title>
 <title>&productname; Documentation</title>

```

Reply:
@sknorr I guess with your latest commit, we can consider this resolved and fixed?

Reply:
Yes.